### DISCUSSION TITLE
[@types/koa] Wrong typing for koa callback function

### DISCUSSION THREAD

**Discussion by crystalfp** [DefinitelyTyped/DefinitelyTyped#64923] [Issues with a @types package]
At line 519 of `@types/koa/index.d.ts` the return type of `callback()` function is wrong: it is `void`, not `Promise<void>`

`@types/koa` is version 2.13.6 and `koa` is version 2.14.1

Thanks for looking!
mario

[a comment by an automated account is not shown]

**Comment by pkuczynski** [comment]
I believe this has been fixed recently: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64809

**Reply by crystalfp** [reply]
I don't call it a fix. Was working before and now I have to silence eslint without any benefit from the change.
